appservice/src/intTest/groovy/nu/yona/server/PinResetRequestTest.groovy

```diff
@@ -72,14 +81,33 @@ class PinResetRequestTest extends AbstractAppServiceIntegrationTest
 		appService.deleteUser(richard)
 	}
 
-	def 'Clear pin reset request before verification'()
+	def 'Try to verify pin reset confirmation code before end of delay period'()
```

Comment:
If the app is built correctly, then it indeed cannot happen, as the link is unavailable. So hacking attempt is indeed a better name. I'll edit that in-place.